diff --git a/magic/Magdir/filesystems b/magic/Magdir/filesystems
index a08b226..a2891dc 100644
--- a/magic/Magdir/filesystems
+++ b/magic/Magdir/filesystems
@@ -1358,7 +1358,7 @@
 >11		uleshort&0x001f	0	
 >>11		uleshort	<32769		
 >>>11		uleshort	>31		
->>>>21		ubyte&0xf0	0xF0		
+>>>>21		ubyte&0xf0	0xF0            x86 boot sector
 >>>>>0		ubyte		0xEB		DOS/MBR boot sector
 >>>>>>1		ubyte		x		\b, code offset 0x%x+2
 >>>>>0		ubyte		0xE9		
