--- file-5.31/magic/Magdir/filesystems.0107~	2017-05-25 13:17:12.913609354 +0200
+++ file-5.31/magic/Magdir/filesystems	2017-05-25 13:18:46.777353051 +0200
@@ -1363,7 +1363,7 @@
 >11		uleshort&0x001f	0
 >>11		uleshort	<32769
 >>>11		uleshort	>31
->>>>21		ubyte&0xf0	0xF0
+>>>>21		ubyte&0xf0	0xF0		x86 boot sector
 >>>>>0		ubyte		0xEB		DOS/MBR boot sector
 >>>>>>1		ubyte		x		\b, code offset 0x%x+2
 >>>>>0		ubyte		0xE9
