diff -up file-5.46/magic/Magdir/rpm.4~ file-5.46/magic/Magdir/rpm
--- file-5.46/magic/Magdir/rpm.4~	2024-11-29 17:00:43.009313041 +0100
+++ file-5.46/magic/Magdir/rpm	2024-11-29 22:24:40.171087227 +0100
@@ -34,6 +34,7 @@
 >>8	beshort		22		RISC-V64
 >>8	beshort		23		LoongArch64
 >>8	beshort		255		noarch
+>>10	string		x		%s
 
 #delta RPM    Daniel Novotny (dnovotny@redhat.com)
 0	string		drpm		Delta RPM
