diff -Naur file-5.28/magic/Magdir/perl file-5.28.tpg/magic/Magdir/perl
--- file-5.28/magic/Magdir/perl	2016-06-12 16:50:16.000000000 +0000
+++ file-5.28.tpg/magic/Magdir/perl	2016-06-20 15:33:48.864971768 +0000
@@ -7,29 +7,41 @@
 # Send additions to <perl5-porters@perl.org>
 0	search/1024	eval\ "exec\ perl		Perl script text
 !:mime	text/x-perl
+!:strength + 20
 0	search/1024	eval\ "exec\ /bin/perl		Perl script text
 !:mime	text/x-perl
+!:strength + 20
 0	search/1024	eval\ "exec\ /usr/bin/perl	Perl script text
 !:mime	text/x-perl
+!:strength + 20
 0	search/1024	eval\ "exec\ /usr/local/bin/perl	Perl script text
 !:mime	text/x-perl
+!:strength + 20
 0	search/1024	eval\ 'exec\ perl		Perl script text
 !:mime	text/x-perl
+!:strength + 20
 0	search/1024	eval\ 'exec\ /bin/perl		Perl script text
 !:mime	text/x-perl
+!:strength + 20
 0	search/1024	eval\ 'exec\ /usr/bin/perl	Perl script text
 !:mime	text/x-perl
+!:strength + 20
 0	search/1024	eval\ 'exec\ /usr/local/bin/perl	Perl script text
 !:mime	text/x-perl
+!:strength + 20
 0	search/1024	eval\ '(exit\ $?0)'\ &&\ eval\ 'exec	Perl script text
 !:mime	text/x-perl
+!:strength + 20
 0	string	#!/usr/bin/env\ perl	Perl script text executable
 !:mime	text/x-perl
+!:strength + 20
 0	string	#!\ /usr/bin/env\ perl	Perl script text executable
 !:mime	text/x-perl
+!:strength + 20
 0	string	#!
 >0	regex	\^#!.*/bin/perl([[:space:]].*)*$	Perl script text executable
 !:mime	text/x-perl
+!:strength + 20
 
 # by Dmitry V. Levin and Alexey Tourbin
 # check the first line
