diff -up file-5.38/src/seccomp.c.9~ file-5.38/src/seccomp.c
--- file-5.38/src/seccomp.c.9~	2019-07-18 22:32:06.000000000 +0200
+++ file-5.38/src/seccomp.c	2020-01-05 02:55:36.515768272 +0100
@@ -181,6 +181,7 @@ enable_sandbox_full(void)
 #ifdef __NR_getdents64
 	ALLOW_RULE(getdents64);
 #endif
+	ALLOW_RULE(getpid);
 #ifdef FIONREAD
 	// called in src/compress.c under sread
 	ALLOW_IOCTL_RULE(FIONREAD);
@@ -222,7 +223,6 @@ enable_sandbox_full(void)
 #if 0
 	// needed by valgrind
 	ALLOW_RULE(gettid);
-	ALLOW_RULE(getpid);
 	ALLOW_RULE(rt_sigtimedwait);
 #endif
 
